encoders/nlp/TransformerTorchEncoder/__init__.py

```diff
@@ -8,170 +8,156 @@
-            - a string with the `shortcut name` of a pre-trained model to load from cache or download, e.g.: ``bert-base-uncased``.
-            - a string with the `identifier name` of a pre-trained model that was user-uploaded to Hugging Face S3, e.g.: ``dbmdz/bert-base-german-cased``.
+            - a string, the `model id` of a pretrained model hosted inside a model repo on huggingface.co, e.g.: ``bert-base-uncased``.
             - a path to a `directory` containing model weights saved using :func:`~transformers.PreTrainedModel.save_pretrained`, e.g.: ``./my_model_directory/``.
-            - a path or url to a `tensorflow index checkpoint file` (e.g. `./tf_model/model.ckpt.index`). In this case, ``from_tf`` should be set to True and a configuration object should be provided as ``config`` argument.
-            This loading path is slower than converting the TensorFlow
-            checkpoint in a PyTorch model using the provided conversion scripts and loading the PyTorch model afterwards.
+        :param base_model: The name of the base model to use for creating the tokenizer. If None, will be equal to `pretrained_model_name_or_path`.
         :param pooling_strategy: the strategy to merge the word embeddings into the chunk embedding. Supported
-            strategies include 'auto', 'cls', 'mean', 'max', 'min'.
+            strategies include 'cls', 'mean', 'max', 'min'.
         :param layer_index: index of the transformer layer that is used to create encodings. Layer 0 corresponds to the embeddings layer
         :param max_length: the max length to truncate the tokenized sequences to.
         :param model_save_path: the path of the encoder model. If a valid path is given, the encoder will be saved to the given path
-        :param truncation_strategy: select truncation strategy. Supported values
-            * `True` or `'longest_first'` (default): truncate to a max length specified in `max_length` or to the max acceptable input length for the model if no length is provided (`max_length=None`).
-            * `'only_first'`:  This will only truncate the first sequence of a pair if a pair of sequences (or a batch of pairs) is provided
-            * `'only_second'`: This will only truncate the second sequence of a pair if a pair of sequences (or a batch of pairs) is provided
-            * `False` or `'do_not_truncate'`: No truncation (i.e. can output batch with sequences length greater than the model max admissible input size)
+        :param truncation_strategy: Truncation strategy
+            ..deprecated::
+                This parameter has no effect - ``truncation=True`` is always applied on tokenizing by default.
 
         ..warning::
             `model_save_path` should be relative to executor's workspace
         """
 
         super().__init__(*args, **kwargs)
         self.pretrained_model_name_or_path = pretrained_model_name_or_path
+        self.base_model = base_model if base_model else pretrained_model_name_or_path
```

Comment:
Not sure, but is possible that `base_model or pretrained_model_name_or_path`  would also work?